webapp/src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { LinearProgress, Tooltip, Typography } from '@mui/material';
+
+const ProgressBar = () => {
+  /**
+   * @privateRemarks
+   * the dates are placeholders for now. The actual date values will be pulled from the database later on
+   * months read as: 0 - 11 instead of 1 - 12
+   * will later add a function to convert the date string into the Date format
+   */
+
+  const start_date = new Date(2022, 4, 1).getTime();
+  const end_date = new Date(2022, 7, 30).getTime();
```

Comment:
Are these values going to be brought in from the database later?

Reply:
Ideally, yes, these are just placeholders for now. I also planned to add a function in the lines above that would parse however the `TIME` data is stored from the database into integers that the `Date` format could use. If I recall, in the database, it would be a stored in a `YYYY-MM-DD` TIMESTAMP string (?), so we would have the function parse this.